docs/solutions/sql_server_always_on_availability_groups/modules/iam/compartment/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "id" {
+  value = "${ join(" ", oci_identity_compartment.compartment.*.id) }" 
```

Comment:
Don't need this join since count is no longer used.

Reply:
good point.  pushing change.

Reply:
Done.